CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+## Setup
+
+We recommend following setup:
```

Comment:
I think a tad more nativespeaky would be "We **do** recommend **the** following setup"

Reply:
J'agree